recipes/nameof/all/test_package/test_package.cpp

```diff
@@ -10,8 +10,8 @@ int main() {
   constexpr auto name = NAMEOF(structvar);
   static_assert("structvar" == name);
 
-  std::string_view res1 = NAMEOF(structvar);
-  std::string_view res2 = NAMEOF(::structvar);
+  auto res1 = NAMEOF(structvar);
+  auto res2 = NAMEOF(::structvar);
```

Comment:
The return type is `cstring`: https://github.com/Neargye/nameof/blob/master/doc/reference.md#nameof

I think this is a case, where string_view was living more than the return type, that's why tests did not pass.